### PR TITLE
[12.0][IMP] web_pwa_oca: Drop BeforeInstallPrompt

### PR DESCRIPTION
**Pull request by Tardo**
BeforeInstallPromptEvent is deprecated. See https://github.com/w3c/manifest/pull/836

Currently only chrome support it... and can't be used for the purpose that is used in the module.

cc @tecnativa TT23822

ping @thomaspaulb

### CODE REVIEW

File: web_pwa_oca/README.rst [hunk 1]
```diff
@@ -92,7 +92,10 @@ Usage
 To use your PWA:
 
 #. Open the Odoo web app using a supported browser (like Chrome/Chromium)
```
Comment (thomaspaulb) [inline web_pwa_oca/README.rst]: Or Firefox.

File: web_pwa_oca/static/description/index.html [hunk 2]
```diff
@@ -433,8 +433,10 @@ <h1><a class="toc-backref" href="#id3">Usage</a></h1>
 <p>To use your PWA:</p>
 <ol class="arabic simple">
 <li>Open the Odoo web app using a supported browser (like Chrome/Chromium)</li>
-<li>Install the PWA</li>
+<li>Open the browser options</li>
+<li>Click on ‘Add to Home screen’ (or ‘Install’ in other browsers)</li>
```
Comment (thomaspaulb) [inline web_pwa_oca/static/description/index.html:436]: Also remove [this readme line](https://github.com/OCA/web/blame/12.0/web_pwa_oca/readme/INSTALL.rst#L13)

**Review by thomaspaulb — APPROVED** [review]
Good find, I didn't know about that. I'm in agreement, it makes the module simpler also.

File: web_pwa_oca/README.rst [hunk 3]
```diff
@@ -91,8 +89,11 @@ Usage
 
 To use your PWA:
 
-#. Open the Odoo web app using a supported browser (like Chrome/Chromium)
-#. Install the PWA
+#. Open the Odoo web app using a supported browser (See https://caniuse.com/?search=A2HS)
+#. Open the browser options
+#. Click on 'Add to Home screen' (or 'Install' in other browsers)
+
+** Maybe you need refresh the page to load the service worker after use 'add to home screen'.
```
Comment (pedrobaeza) [inline web_pwa_oca/README.rst]:
```suggestion
** Maybe you need refresh the page to load the service worker after using the option.
```

**Comment by legalsylvain** [comment]
Hi @Tardo and Tecnativa team. First, thanks a lot for this usefull module, I use since a month in production.
Recently (I guess since https://github.com/OCA/web/pull/1628) I have a problem with Firefox + private mode. 

i so made a patch on the top of your current PR to fix it. https://github.com/Tecnativa/web/pull/2
Could you merge it, if it's OK for you ?

(I PR against your PR to avoid conflict, because you are changing the line I want to change).

thanks !

**Comment by Tardo** [comment]
Changes done

**Review by pedrobaeza — APPROVED** [review]

**Review by marcelsavegnago — APPROVED** [review]

**Pull request merged**